### ISSUE TITLE
Greg Sutcliffe (again :P) - stuff

### ISSUE THREAD

**Issue by GregSutcliffe** [techmeetup/edinburgh#113]
Heya. Was chatting with Rachel and she seemed interested in a talk I was giving down south on remote working - upsides, downsides, employee and employer perspectives, etc. Would be happy to do that for tech meetup if you find yourself short a speaker again ;)

Alternatively, I'm doing one next month on building open source communities. If it goes well, you can have a version of that :p

Cheers
Greg

**Comment by GregSutcliffe** [comment]
Oh, for the record, can't do December. After that, all good.

**Comment by marksutherland** [comment]
Hi Greg,

That sounds great :) I think we're probably going to have some lightning talks in January, so the earliest would probably be February, but I'll get back in touch closer to the time. Hope you're doing well!

**Comment by GregSutcliffe** [comment]
February 8th is out, I'll be away on conference. Match onwards is fine.

Oh, and you can add Nextcloud (http://nextcloud.com/) and general stuff on privacy and taking control of your own data as a potential topic. Using myself as a guinea pig, I'm currently running Android with no Google content and it's working well thanks to Nextcloud :P

**Comment by marksutherland** [comment]
Hi @GregSutcliffe , we're a little shorter on speakers for the next few months, so if you're available that would be great. Dates are May 10th, June 14th and July 12th. FYI, we have another potential speaker who's offering to talk about remote working, so either open source communities or next cloud would be good, whichever you're happier to talk about :) Thanks again for volunteering!

**Comment by GregSutcliffe** [comment]
Life complications make May/June very difficult. July 12th is possible. I'll drop a TODO on my list to think about topics, and get back to you in a week or so if that's ok?

**Comment by marksutherland** [comment]
Hi Greg, that's fine. Let me know what you think!

**Comment by GregSutcliffe** [comment]
So I think Nextcloud works best. How about I talk about my personal journey of the last year or so, in moving away from the big data providers (Google, for the most part in my case, but you could also include Apple, Microsoft, Facebook, etc) towards owning my own data again?

**Comment by marksutherland** [comment]
Hi Greg,

That sounds good to me! I'll pencil you in for July. Thanks again! :)

Mark

**Comment by marksutherland** [comment]
Hi Greg,

Just wanted to confirm you're good for July 12th? Assuming yes, any questions or concerns? And do you have a title? Thanks!

Mark

**Comment by GregSutcliffe** [comment]
Yes, I'm still good for the 12th. No concerns, I've spoken at TMU enough times to know what to expect - you have VGA input, right?

Title, how about "Giving Back The Free Lunch"
Subtitle "How & why to handle your own data"
Abstract "Over the last year, I've been busy trying to take back control of various facets of my data. In this talk I'll cover what I've been working on (mostly de-Googling my Android phone, but some other stuff too), why I think it's important, and how you can go about it"

Sounds good?

**Comment by marksutherland** [comment]
I don't think we do have VGA, just HDMI and mini display port.

Talk sounds good to me, thanks for letting me know!

**Comment by GregSutcliffe** [comment]
Ah, OK, I'll make sure I have something compatible & a PDF backup :P
